packages/azure-services/src/azure-queue/queue.ts

```diff
@@ -84,6 +91,18 @@ export class Queue {
         return queueProperties.approximateMessagesCount;
     }
 
+    private async tryCreateMessage(queue: string, message: unknown): Promise<boolean> {
+        const queueURL = await this.getQueueURL(queue);
+        await this.ensureQueueExists(queueURL);
+        const enqueued = await this.createQueueMessage(queueURL, message);
+
+        if (enqueued) {
+            return true;
+        } else {
+            throw new Error(`Failed to enqueue the message`);
```

Comment:
add the util.inspect(message) to the error object

Reply:
there is no error object here.

Reply:
i meant the message, so that you know for which message the enqueuing failed.

Reply:
Added.